### ISSUE TITLE
Just a report on ip banning

### ISSUE THREAD

**Issue by ptptp10086** [finndev/PokeBuddy#817]
So i tried to locate my own. im in USA, i use to farm in Los Angeles  i live 1 hour away from boston. after the frequent 15mins ip ban, i tried to switch my farming spot to boston commons. im still getting ip bans, so im guessing this is not a gps ip thing, because i am using USA ip or are they checking the exact ip? i do live an hour away from boston so the city ip is diffrent could that be the issue?

**Comment by drdetroit1** [comment]
Ran in central park new york for a couple hours im in michigan with no issues other then the access token expiring and turning stops back to blue every 10 seconds

**Comment by m0r73n** [comment]
I don't think that it is a GPS thing either. I'm farming in the US but have an European IP. No problem in the emulator in NOX which works perfectly. But the bot here gets IP-banned constantly at the same GPS spot as in NOX. Changing IP works for a short while but then I get an IP-ban again. And I only have one account - no teleporting at all - speed at 10 km/h - no evolving or hatching eggs.

**Comment by r0nalxd** [comment]
For more info, check my post in issue #707
